src/main/resources/application.yaml

```diff
@@ -147,7 +147,7 @@ uk:
             SERVICE_APPLICATION_REJECTED: 'db7f376b-ab61-4316-b59f-83fe570b8f23'
             SERVICE_APPLICATION_GRANTED: '2b7cc04e-2c21-4c06-9ae4-e7109a3efde4'
             RESPONDENT_SOLICITOR_HAS_NOT_RESPONDED: 'e872dbc4-e1f9-454c-830c-3e86ccb03251'
-            APPLY_FOR_FINAL_ORDER_SOLICITOR: '70dd90fe-e702-4daa-8f69-4ab41518d9a6'
+            APPLY_FOR_FINAL_ORDER_SOLICITOR: '91c049a6-d2ee-4682-bea2-8a74fa1b75a6'
```

Comment:
should we also update the welsh side? just in case even tho its not translated, the old template was also used in the welsh even tho it was in english.

Reply:
Yeah good q - it's been signed off so I assume this piece of work is coming later?

Reply:
I dont know but I am pretty sure they dont test welsh when signing off. I think its fine until you delete the old template.